### ISSUE TITLE
Repacking files with whitespace in filenames

### ISSUE THREAD

**Issue by mjarr** [nunuhara/alice-tools#28]
I'm trying to repack Evenicle's CGs with ALICEPACK, but the program throws a syntax error when it encounters a line with a space in the filename. There are 3 such files EvenicleCG.afa.

Edit:
I dealt with this issue myself, but looks like archives created with pack are not compatible with Evenicle either way (main menu is all black). I'm using linux binary built from patched master.

Edit2:
I managed to pack it with BATCHPACK after hacking it to accept dcf and webp files. Either ALICEPACK is broken or I'm using it incorrectly.

**Comment by nunuhara** [comment]
Thanks for the report. I have fixed this in the latest release (0.10.1) by allowing filenames to be quoted in manifest files. E.g.

    #ALICEPACK
    test.afa
    "white space.qnt"

**Comment by mjarr** [comment]
Encoding seems to be completely broken now. Both windows binary and debian alice compiled from master throw
`ERROR: can't determine size of file: outjp_raw/�o�f�s�^�o���j���[�^�ėp�y��.qnt`

My manifest looks like this by the way:
```
#ALICEPACK
EvenicleCG.afa
"outjp_raw/ＰＧＴ／Ｐメニュー／汎用土台.qnt"
"outjp_raw/ＰＧＴ／アイテム使用／キャラ選択不可.qnt"
"outjp_raw/ＰＧＴ／システム／Pボタン／白／汎用.qnt"
...
```

**Comment by nunuhara** [comment]
This should be fixed now in 0.10.2

**Comment by mjarr** [comment]
Yay, repacking works now.
However archives packed with ALICEPACK still are not loaded by the game. Should I make a separate issue?
